src/test/regress/expected/multi_cluster_management.out

```diff
@@ -882,6 +1049,15 @@ SELECT * FROM pg_dist_node WHERE nodeid = :worker_1_node;
      17 |      14 | localhost |    57637 | default  | f           | t        | primary  | default     | f              | t
 (1 row)
 
+SET client_min_messages TO ERROR;
+SELECT start_metadata_sync_to_node(nodename, nodeport) FROM pg_dist_node WHERE isactive = 't' and noderole = 'primary';
+ start_metadata_sync_to_node
+---------------------------------------------------------------------
+
+
+(2 rows)
+
+RESET client_min_messages;
```

Comment:
I think at the end of this file, we should have a test which verifies that all the primary nodes have metadata synced:
```SQL
-- verify that at the end of this file, all primary nodes have metadata synced
SELECT bool_and(hasmetadata) & bool_and(metadatasynced) FROM pg_dist_node WHERE isactive = 't' and noderole = 'primary';
```

This file is way too complex, and we should guard for future changes.